sos/plugins/insights.py

```diff
@@ -13,16 +13,16 @@ class RedHatInsights(Plugin, RedHatPlugin):
     '''Collect config and log for Red Hat Insights
     '''
     plugin_name = 'insights'
-    packages = ['redhat-access-insights']
+    packages = ['insights-client']
     profiles = ('system', 'sysmgmt')
-    conf_file = '/etc/redhat-access-insights/redhat-access-insights.conf'
+    conf_file = '/etc/insights-client/insights-client.conf'
 
     def setup(self):
         self.add_copy_spec(self.conf_file)
         if self.get_option("all_logs"):
-            self.add_copy_spec("/var/log/redhat-access-insights/*.log*")
+            self.add_copy_spec("/var/log/insights-client/*.log*")
         else:
-            self.add_copy_spec("/var/log/redhat-access-insights/*.log")
+            self.add_copy_spec("/var/log/insights-client/*.log")
 
```

Comment:
Likewise (and since these are just log files passed to `add_copy_spec()` you can just add them all: the plugin will add those that exist and ignore the rest.

Reply:
The old log file path does not exist. 

Should I remove the "else" part to collect all insight logs?

Reply:
This:

```python
            self.add_copy_spec("/var/log/insights-client/*.log*")
        else:
self.add_copy_spec("/var/log/insights-client/*.log")
```

Is incorrect - the two branches collect the exact same data. One should collect exactly one log file (the current), e.g. `insights.log` if that's the file name used, and the other collects "all" logs, using the `*.log` glob (this bug was already present in the plugin before your changes).